src/Shared/UnitTests/MockEngine.cs

```diff
@@ -198,14 +154,10 @@ internal string Log
 
                 _log.Clear();
             }
-            get { return _log.ToString(); }
+            get => _log.ToString();
```

Comment:
It seems weird that the getter is after the setter, do you mind fixing that?  Its like that in this whole file for some reason

Reply:
Good catch. Reversed in whole file